Jenkinsfile

```diff
@@ -333,17 +319,14 @@ pipeline {
         axes {
           axis {
             name 'CASSANDRA_VERSION'
-            values '2.1',     // Legacy Apache Cassandra
-                   '3.11',    // Current Apache Cassandra
+            values '3.11',    // Current Apache Cassandra
                    '4.0',     // Development Apache Cassandra
                    'dse-5.1.35', // Legacy DataStax Enterprise
-                   'dse-6.0.18', // Previous DataStax Enterprise
-                   'dse-6.7.17', // Current DataStax Enterprise
                    'dse-6.8.30' // Development DataStax Enterprise
           }
           axis {
             name 'NODEJS_VERSION'
-            values '8', '10', '12'
+            values '8', '10', '12', '16', '18', '20'
```

Comment:
Along the lines of what's discussed above 8, 10 and 12 would be removed here as well.